src/main/java/core/basesyntax/strategy/IncrementHandler.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.StorageService;
+
+public class IncrementHandler implements OperationHandler {
+    private final StorageService storageService;
+
+    public IncrementHandler(StorageService storageService) {
```

Comment:
The `IncrementHandler` class is tightly coupled with `StorageService`. Consider following the SOLID principles to make classes more independent. Instead of directly using `StorageService`, you might want to use an interface that `StorageService` implements, allowing for more flexibility and easier testing.